lib/worker.js

```diff
@@ -121,6 +141,10 @@ Worker.prototype.run = function () {
     queue: this.queue,
     job: this.job
   }
+  if (process.env.WORKER_MONITOR) {
```

Comment:
I don't think for us there is a case where we don't want to get this data. Might be better to check for DATADOG_HOST